requirements-dev.txt

```diff
@@ -3,8 +3,7 @@ pandas>=0.23
 pyarrow>=0.10
 numpy>=1.14
 
-# TODO: support mlflow 1.0.0
-mlflow==0.9.1
+mlflow[extras]>=1.0
```

Comment:
For context: We made a lot of breaking changes in 1.0. Most apis (those not marked experimental) are going to be stable from this point on.

Reply:
I see, sounds good.
btw, could you add the "extras" libraries in the `# Test` section below instead of using `mlflow[extras]`? This file is also used by `conda` which doesn't handle the extra notation.
Also, could you update `setup.py` file?